source/channels/format-messages.rst

```diff
@@ -128,7 +122,11 @@ The message formatting toolbar offers the following formatting options:
 
 Review how your message formatting will look when the message is sent by selecting the **Show/Hide Preview** |preview-icon| icon. Return to your draft message by selecting the icon again.
 
-You can hide the formatting options by selecting the **Show/Hide Formatting** |hide-formatting-icon| icon. Select the icon again to show the formatting options.
+.. tip::
+
+  - Hide the formatting options by selecting the **Show/Hide Formatting** |hide-formatting-icon| icon. Select the icon again to show the formatting options. 
+  - Disable the formatting options by selecting the **Gear** |settings-icon| icon, going to **Settings > Advanced > Enable Post Formatting**, and setting this option to **Off**. 
+  - System Admins can disable the formatting options for all users in the System Console by going to **Environment > Developer > Enable Client Performance Debugging**. See the `configuration settings <https://docs.mattermost.com/configure/environment-configuration-settings.html#enable-client-performance-debugging>`__ documentation for details.
```

Comment:
This hasn't been changed yet, so it's not accurate. That setting is currently available all of the time

Reply:
@hmhealey - My understanding of the configurability of this feature may be incorrect. I thought we had:
- a way to show/hide the formatting options within the UI element itself (the eye icon)
- a way for users to disable it via Settings so they never see it again
- a server-side kill switch in the System Console whose current label isn't as discoverable as we'd like

Which parts of my understanding are incorrect?

Reply:
As above, I think we're getting some stuff confused here. For the post textbox:
- The eye button in the post textbox for turning on and off the formatting toolbar on it.
- I think there's a server-side setting for turning the formatting toolbar on and off entirely. Presumably this is going away eventually since we want everyone using it.

That's separate from the Enable Post Formatting option in the Advanced settings. We did also discuss moving it behind the Enable Client Performance Debugging setting since it's only intended for debugging, but that hasn't been done yet

Reply:
Thank you for the clarifications! The misunderstandings are entirely mine :)